feature/experimental/gribi/ate_tests/route_addition_during_failover_test/route_addition_during_failover_test.go

```diff
@@ -134,6 +135,12 @@ func coreFileCheck(t *testing.T, dut *ondatra.DUTDevice, gnoiClient raw.GNOI, sy
 		Path: vendorCoreFilePath[dut.Vendor()],
 	}
 	validResponse, err := gnoiClient.File().Stat(context.Background(), in)
+	if err != nil {
+		if retry {
```

Comment:
So we only retry it once? Is it sufficient?

Reply:
yes, it should recover with one retry.